content/source/docs/cloud/vcs/bitbucket-server.html.md

```diff
@@ -1,17 +1,19 @@
 ---
 layout: "cloud"
-page_title: "Bitbucket Server - VCS Providers - Terraform Cloud"
+page_title: "Bitbucket Server and Data Center - VCS Providers - Terraform Cloud"
 ---
 
 
-# Configuring Bitbucket Server Access
+# Configuring Bitbucket Server/Data Center Access
 
-These instructions are for using Bitbucket Server for Terraform Cloud's VCS features. Bitbucket Server is the on-premise version of Bitbucket; [Bitbucket Cloud has separate instructions,](./bitbucket-cloud.html) as do the [other supported VCS providers.](./index.html)
+These instructions are for using Bitbucket Server for Terraform Cloud's VCS features. Bitbucket Server is the on-premises version of Bitbucket; [Bitbucket Cloud has separate instructions,](./bitbucket-cloud.html) as do the [other supported VCS providers.](./index.html)
 
 Note that Bitbucket Server requires both OAuth authentication and an SSH key. The instructions below include SSH key configuration.
 
 -> **Version note:** Terraform Cloud supports Bitbucket Server versions 4.9.1 and newer. HashiCorp does not test older versions of Bitbucket Server with Terraform Cloud, and they might not work as expected. Also note that, although we do not deliberately remove support for versions that have reached end of life (per the [Atlassian Support End of Life Policy](https://confluence.atlassian.com/support/atlassian-support-end-of-life-policy-201851003.html)), our ability to resolve customer issues with end of life versions might be limited.
```

Comment:
```suggestion
-> **Version note:** Terraform Cloud supports Bitbucket Server versions 4.9.1 and newer, and Bitbucket Data Center versions 5.4.0 and newer. HashiCorp does not test older versions of Bitbucket Server with Terraform Cloud, and they might not work as expected. Also note that, although we do not deliberately remove support for versions that have reached end of life (per the [Atlassian Support End of Life Policy](https://confluence.atlassian.com/support/atlassian-support-end-of-life-policy-201851003.html)), our ability to resolve customer issues with end of life versions might be limited.
```

Reply:
I think this is actually confusing as it doesn't draw a distinction between Bitbucket Server (legacy) and Bitbucket Server (native - new) - we have two distinct repo types now, because of technical differences between them - and we have explicit version statements for both, one of which allows us to use Data Center and one of which does not. I think the prior support statement I wrote was clearer as it laid.

Reply:
I'm not sure what you mean here; the rest of the docs don't draw a named distinction between the two, and the UI when creating the connection only asks whether your version number is over the mark. 

The reason I found your original wording confusing was this part: 

> Bitbucket Data Center users will follow the same instructions as a user with Bitbucket Server **with version greater than v5.4.0**, omitting the installation of the Web Post Hooks add-on.

That reads as "any BBDC user should behave as though they were on BBS version 5.4+," when in fact it's sort of the opposite: BBDC users must be on version 5.4+, and then should just follow the appropriate instructions.

Reply:
OK, I can see how you would think that. This seems OK to me if you are satisfied with it. Given this much more streamlined wording, do you still want to change the title of the page and sidebar nav link or no?

Reply:
Yeah, go ahead and leave them as "server and data center." That way users won't have any doubt about which page they need to go to, and once they're there they should be able to understand the situation.